api/v2/checluster_types.go

```diff
@@ -479,6 +479,12 @@ type PersistentHomeConfig struct {
 	// Must be used with the 'per-user' or 'per-workspace' PVC strategy in order to take effect.
 	// Disabled by default.
 	Enabled *bool `json:"enabled,omitempty"`
+	// Determines whether the init container that initializes the persistent home directory should be disabled.
+	// When the `/home/user` directory is persisted, the init container is used to initialize the directory before
+	// the workspace starts. If set to true, the init container will not be created.
+	// This field is not used if the `workspace.persistUserHome.enabled` field is set to false.
+	// Enabled by default.
```

Comment:
I'm a bit confused, I thought this was [disabled by default](https://github.com/devfile/devworkspace-operator/pull/1251/files#diff-2658370b0497fa11f0682953637e6e2d237a568851f410b6de8e1ad202045a6cR48)? Do we have a docs bug in DWO?

Reply:
Right, the value is set to false by default, but the init container itself enabled by default

Reply:
Ah okay that makes more sense. Maybe we should write "Set to false by default" or "The init container is enabled by default"? I'm okay with leaving it as is, but it almost reads as "DisableInitContainer is enabled by default = the init contianer is disabled by default"